### DISCUSSION TITLE
for commandline plugin is it possible or even wise to show the current directory next to the data-host?

### DISCUSSION THREAD

**Discussion by simkimsia** [PrismJS/prism#3327] [Q&A] [answered]
<img width="842" alt="image" src="https://user-images.githubusercontent.com/245021/153363565-51845f8a-8a64-44fe-a739-5c68b0642531.png">

This is my commandline usage.

As you can see it's always `$` which is correct for non root user, but my question is it wise or even possible to **also show** where the current working directory is? since i have a `cd` for navigation involved.

**Comment by RunDevelopment** [comment, marked as answer]
As [documented](https://prismjs.com/plugins/command-line/), you can use the `data-prompt` to specify any text you want. However, it's always the same text, so you can't emulate `cd` with Command Line.

However, you might be interested in the [Shell sessions language](https://prismjs.com/test.html#language=shell-session&text=foo%40bar%3A%2F%24%20cd%20~%0Afoo%40bar%3A~%24%20sudo%20-i%0A%5Bsudo%5D%20password%20for%20foo%3A%0Aroot%40bar%3A~%23%20echo%20%22hello!%22%0Ahello!). This language highlights (ba)sh terminal text.
